Classes/Emogrifier.php

```diff
@@ -190,25 +210,13 @@ public function emogrify()
         if ($nodesWithStyleAttributes !== false) {
             /** @var \DOMElement $node */
             foreach ($nodesWithStyleAttributes as $node) {
-                $normalizedOriginalStyle = preg_replace_callback(
-                    '/[A-z\\-]+(?=\\:)/S',
-                    function (array $m) {
-                        return strtolower($m[0]);
-                    },
-                    $node->getAttribute('style')
-                );
-
-                // in order to not overwrite existing style attributes in the HTML, we have to save
-                // the original HTML styles
-                $nodePath = $node->getNodePath();
-                if (!isset($this->styleAttributesForNodes[$nodePath])) {
-                    $this->styleAttributesForNodes[$nodePath] = $this->parseCssDeclarationBlock(
-                        $normalizedOriginalStyle
-                    );
-                    $this->visitedNodes[$nodePath] = $node;
+                if ($this->isInlineStylesParsingEnabled) {
+                    // Normalize the "style" attribute and save it for later retrieval
```

Comment:
I don't think that we need all these comments above the method calls if the methods have good, speaking names and the method PHPDoc are good.